src/contents/_rounding.scss

```diff
@@ -154,3 +155,29 @@
     }
   }
 }
+
+@include Option("userChrome.rounding.square_infobox") {
+  @include moz-document(
+    url-prefix "about:",
+    regexp "^chrome:\/\/\\w+\/content\/.*\.xhtml$"
+  ) {
+    /*
+     * .container is inside shadow DOM,
+     * and it does not have any other classes,
+     * but there may be other elements with the same class,
+     * hence the weird selector.
+     */
+    link[href="chrome://global/content/elements/message-bar.css"] + .container,
+    .info-box-container:not(.content-blocking-category),
```

Comment:
This is just a question.
Does Messagebar appear on the web content? Or is it another elements?

Reply:
it will appear in about:addons if you delete an extension

Reply:
It has been confirmed. Thank you for waiting for a long time.
![image](https://user-images.githubusercontent.com/25581533/232749352-addf94d5-41af-4628-812b-25b0c0926797.png)